extensions/vscode/src/commands/new-middleware.ts

```diff
@@ -20,9 +21,11 @@ import {
  *
  * This command is available from the command palette and the context menu.
  *
- * When launching the command from the command palette, the Uri is undefined
- * and the user is prompted to select a valid directory or file to create the
- * route in.
+ * When launching the command from the command palette, the Uri is undefined.
+ * Therefore, the command attempts to resolve a path from the user's active text
```

Comment:
```suggestion
 * Therefore, the command attempts to resolve a path from the user's active
```

Reply:
Going to keep "active text editor" since the VS Code API uses such terminology (window.activeTextEditor)